packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -337,8 +339,11 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     filteredLatestErrors = filteredLatestErrors.filter((e) => e.type !== 'unableToStartVideoEffect');
   }
 
+  /* @conditional-compile-remove(acs-close-captions) */
+  const isTeamsCaptions = useSelector(getCaptionsKind) === 'TeamsCaptions';
   /* @conditional-compile-remove(close-captions) */
-  const isTeamsCall = useSelector(getIsTeamsCall);
+  const isTeamsCall =
+    useSelector(getIsTeamsCall) || /* @conditional-compile-remove(acs-close-captions) */ isTeamsCaptions;
```

Comment:
Is this true? We can be in an ACS call but with a Teams captions bot right?

Reply:
This method here is in stable useSelector(getIsTeamsCall) but now we have a better way to check it by using isTeamsCaptions, however it's in beta only. So we have this here until we can stablize ACS captions. 

Also being in an ACS call but with teams captions bot will only happen when user inside an acs call start calling a teams user, this is not currently supported by the composite.

Reply:
This logic also contradicts other places we check isTeamsCall:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/8e8ea957-01fa-48b2-8f8d-667d6c5ca735)

Can you use `isTeamsCaptions` instead where needed and avoid changing this?

Reply:
> but now we have a better way

Oh nice! Can we move this new logic into the `getIsTeamsCall` selector then so all uses pick up the logic?

Reply:
If no one has started captions - do we still get a captions bot kind?

Reply:
We do but we only get it when the call is connected, so we can't use this logic everywhere. I changed it here because we only use this prop for captions. Maybe I should name it isTeamsCaptions instead?

Reply:
Sounds good!

Reply:
Updated!